README.md

```diff
@@ -1,6 +1,6 @@
 # linters-config
 
-## How to use this repo?
+## How to use this repo? 🤔
```

Comment:
[OPTIONAL] you can change this emoji to

* but it is not required to approve this project.*